config/resources/domain.lisp

```diff
@@ -33,14 +33,21 @@
   :on-path "files")
 
 (define-resource bpmnElement ()
-  :properties `((:name :string ,(s-prefix "bbo:name"))
-                (:classification :uri ,(s-prefix "rdf:type")))
+  :properties `((:name :string ,(s-prefix "bbo:name")))
   :has-many `((process :via ,(s-prefix "teamingAI:belongsToProcess")
                        :as "processes"))
+  :has-one `((bpmnElementType :via ,(s-prefix "org:classification")
```

Comment:
Now that I think about it, from the comment of Boris:
```
I would not use the property org:classification here though. This has a specific meaning - Indicates a classification for this Organization within some classification scheme. You’ll probably need to invent a new property instead (I’m not aware of an existing property which could be reused here. And fwiw, the organization ontology also mentions explicitly it’s also permissible to use sub-classes as a means to represent organizational categories - just to say that this issue about choosing sub-classes or a concept scheme returns in many places, and usually both approaches are ok.
```

Did you agree with Boris the `org:classification` approach is fine? unfortunately I already merged the PR in bpmn-service

Reply:
You we right, that's something I overlooked. After consulting Boris again, we agreed to use `dct:type` instead --> the should be taken care of now

Reply:
New pr for bpmn service: https://github.com/lblod/bpmn-service/pull/6